server/lib/src/filter.rs

```diff
@@ -952,6 +967,10 @@ impl FilterComp {
                 // Pretty hard to mess this one up ;)
                 Ok(FilterComp::SelfUuid)
             }
+            FilterComp::Invalid(attr) => {
+                // This is a special case, we can't validate it.
```

Comment:
Technically this is valid, since we are saying that the partial value was invalid, so the pv validation is done and failed, so we need a valid filter part that will always fail.

Reply:
In otherwords, change the comment to say "While the filter component is named invalid, the invalid here only means that the partial value content was invalid. However, it's still *valid* to proceed with this search since the assertion can never match. This allows queries with invalid pvs to proceed and still evaluate correctly such as in an OR condition. "